public/css/main.css

```diff
@@ -52,7 +52,7 @@ body {
 
 .header .container {
 	position: relative;
-	width: auto;
+	max-width: none;
```

Comment:
With this, you remove the max-width of the header. So it takes the whole space and so its width is equal to the width of the screen